components/scripts/lib/build_scan.sh

```diff
@@ -187,6 +187,8 @@ fetch_and_read_build_scan_data() {
   debug ""
 
   header_row_read=false
+  idx=0
```

Comment:
Will the value of idx ever be something other than 0?

Reply:
This just initializes the variable to 0. It's incremented later on in the loop.  If we don't initialize the variable to zero, then I'm not sure what the behavior will be, as it will have an undefined value when we increment it.

Reply:
I had overlooked the incrementing in the loop. All clear now. Thanks, @jthurne.